.gitignore

```diff
@@ -7,3 +7,4 @@ etc/
 .env
 Gemfile.lock
 tips-and-tricks.md
+tags
```

Comment:
I'd rather not change our .gitignore for personal stuff. You might want to consider adding `tags` to a global .gitignore file on your machine.